conf/app.routes

```diff
@@ -2,3 +2,4 @@
 POST       /traders/:eori/profile                   uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.ProfileController.setProfile(eori: String)
 GET        /customs/traders/goods-profiles/:eori    uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.ProfileController.getProfile(eori: String)
 HEAD       /traders/:eori/profile                   uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.ProfileController.doesProfileExist(eori: String)
+GET        /traders/:eori/records                   uk.gov.hmrc.tradergoodsprofilesdatastore.controllers.GetRecordsController.getRecords(eori: String, lastUpdatedDate:Option[String], page: Option[Int], size: Option[Int])
```

Comment:
Id say take page and option out of the request here and hard code it, we will only ever need one value for these params for the FE so it can be a constant

Reply:
I also don't think last updated is something the FE will send, but rather something we lookup in our db for the latest value and use that in the request, it can also be a constant for nw